docs/commands/preview-docs.md

```diff
@@ -1,39 +1,98 @@
 # `preview-docs`
 
-Preview the API reference docs on your local machine.
+## Introduction
 
-If you have a license key, you will have a preview of the premium Redocly API reference docs. The [`login`](./login.md) command also generates a preview of the premium Redocly API reference docs.
+With this command, you can preview the API reference docs on your local machine.
 
-Otherwise, you'll get a preview of Redoc community edition.
+If you have a license key, you will get a preview of the premium Redocly API reference docs. If you don't, you will get a preview of Redoc community edition.
 
+:::success Tip
+To preview docs using the premium Redocly API reference docs, you must authenticate to the API registry first via the [`login`](./login.md) command.
+:::
 
-### `preview-docs` usage
+## Usage
 
+```bash
+openapi preview-docs <entrypoint>
+openapi preview-docs <entrypoint> [--config=<path>] [--port=<value>]
+openapi preview-docs <entrypoint> [--force] [--help] [--version]
+openapi preview-docs <entrypoint> --version
+```
+
+## Options
+
+option                    | type      | required?    | default     | description
+--------------------------|:---------:|:------------:|:-----------:|------------
+`entrypoint`              | `string`  | yes          | -           | Path to the API definition filename that you want to generate preview for. (See [the section below](#entrypoints) for more options)
+`--config`                | `string`  | no           | -           | Specify path to the config file
+`--force`, `-f`           | `boolean` | no           | -           | Generate preview output even when errors occur
+`--help`                  | `boolean` | no           | -           | Show help
+`--port`, `-p`            | `number`  | no           | 8080        | Preview port
```

Comment:
question: Are there any restrictions related to ports; which ports are supported? Can users put just any number here?

Reply:
These are TCP ports on the computer: 0-65535. https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

0-1023 are system ports.

You can use a port as long as it isn't being used by something else.